packages/components/src/components/post-closebutton/post-closebutton.scss

```diff
@@ -0,0 +1,7 @@
+@use '@swisspost/design-system-styles/mixins/utilities';
+
+post-closebutton button post-icon {
+  @include utilities.high-contrast-mode() {
+    color: LinkText !important;
```

Comment:
The close button is a button not a link, you should therefore use ButtonFace and ButtonText so that HCM users easily understand it. Can you please update the whole PR taken this into account?